### PR TITLE
Have install.sh put the full path to ipfs binary if detected

### PR DESCRIPTION
**Pull request by jedahan**
For some reason, I had to sudo launchctl load, after doing sudo chmod 600 ~/Library/LaunchAgents/io.ipfs.ipfs-daemon.plist and sudo chown root ~/Library/LaunchAgents/io.ipfs-daemon.plist on el capitan.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 32343e80de2a1c37e1f4e90adb5b98e2f46f3987
  * Invalid signoff. Commit message must end with  
    **License: MIT  
Signed-off-by: .* <.*>**

[Guidelines](https://github.com/ipfs/community/blob/master/contribution-guidelines.md#commit-messages) and [a script](https://github.com/ipfs/community/blob/master/dev/tools/hooks/setup_commit_msg_hook.sh) are available to help. Your feedback on GitCop is welcome on [this issue](https://github.com/ipfs/infrastructure/issues/23).

---
This message was auto-generated by https://gitcop.com

**Comment by jbenet** [comment]
Interesting. this SGTM. 

> For some reason, I had to sudo launchctl load, after doing sudo chmod 600 ~/Library/LaunchAgents/io.ipfs.ipfs-daemon.plist and sudo chown root ~/Library/LaunchAgents/io.ipfs-daemon.plist on el capitan.

should the docs be changed to reflect what you had to do?

> There were the following issues with your Pull Request

for OSS hygiene we need the commit to have a sign-off, it's super quick to do, just add the following to your commit message with `git commit --amend`:

```
License: MIT
Signed-off-by: Jonathan Dahan
```
or use this script to setup a hook that does it automatically: https://github.com/ipfs/community/blob/master/dev/tools/hooks/setup_commit_msg_hook.sh

---

@chriscool gitcop is a bit unfriendly still, could we maybe have its response include what i said above (the why -- OSS hygiene -- and the quick one-off how)

**Comment by chriscool** [comment]
@jbenet ok, I will have a look at making Git Cop more friendly soon.

**Comment by jedahan** [comment]
So El Capitan (and supposedly Yosemite) rewrote launchctl and how it works. For sake of backwards compat, I only changed how the daemon is installed if you are on the latest of El Capitan. Wish there was a way to do this without sudo, but I haven't found one so far ( Tried `launchctl bootstrap user/$UID "$dest_dir/$plist"` but that didn't work and the console error message for it was not helpful).

File: misc/launchd/install.sh [hunk 1]
```diff
@@ -19,5 +23,10 @@ if [ $? ]; then
 fi
 
 echo Loading ipfs-daemon
-launchctl load "$dest_dir/$plist"
+if [[ `uname -r` == "15.2.0" ]]; then
```
Comment (jbenet) [inline misc/launchd/install.sh]: this will get out of date soon, maybe just check that the first version is `>= 15`?

**Comment by chriscool** [comment]
@jbenet about GitCop I used to have a Github IPFS login that I used to login into GitCop and to set it up for go-ipfs, but it looks like GitHub has changed the way organisations are managed or something and I cannot login to GitHub as IPFS anymore, so I cannot set up GitCop config for go-ipfs.

**Comment by jbenet** [comment]
@chriscool the account is ipfsbot, the pw should be in meldium-- if it doesnt work, open an issue in https://github.com/ipfs/ops-requests/

**Pull request merged**